dothething/dothething/Assets.xcassets/Placeholder.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "obama.jpg",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
The changes in this file add a new image set named "Placeholder" with an "obama.jpg" image for different scales (1x, 2x, and 3x). This seems to be a placeholder image for the application, likely used as a fallback when a real thumbnail is not available.

The changes look appropriate and follow the standard structure for defining image sets in Xcode. No issues are identified in this file.